packages/expect/src/matchers.js

```diff
@@ -45,14 +45,13 @@ const matchers: MatchersObject = {
 
     const message = pass
       ? () =>
-          matcherHint('.not.toBe', undefined, undefined, {
+          matcherHint('.toBe', undefined, undefined, {
```

Comment:
I wonder why this matcher has asymmetric `toBe`/`not.toBe` output – I don't see any reason we shouldn't suggest toEqual for `not.toBe` assertions, same with showing a diff.
It's not related to this PR, but why not fix it right away.

Reply:
Let’s please wait for a future PR, because it’s a double negative when `.not.toBe` fails.